cmd/incus-agent/exec.go

```diff
@@ -202,6 +202,7 @@ func (s *execWs) Do(op *operations.Operation) error {
 	logger.Debug("Waiting for exec websockets to connect")
 	select {
 	case <-s.requiredConnectedCtx.Done():
+		//nolint:revive //whyNoLint: this is intentional, the flow should continue if all websockets are connected
```

Comment:
Line break after this one

Reply:
92a085656c82d703c86f1f6c42759bf8a6f25dcf